.env-devel

```diff
@@ -152,6 +152,10 @@ POSTGRES_PASSWORD=adminadmin
 POSTGRES_PORT=5432
 POSTGRES_USER=scu
 
+POSTGRES_READONLY_PASSWORD=readonly
```

Comment:
This will require PRs that also create this new env-var in the ops-repos, make sure to merge them together

Reply:
@mrnicegyu11 
- the sister PR is already indicated in the OPS section : https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/938
- After the input by @YuryHrytsuk , this PR does NOT change any docker-compose anymore. For the moment I am just adding the scripts and the env vars but these are for the moment not applied
- I will follow up with ops what would be the most systematic way to create/update and delete this user.

Reply:
@mrnicegyu11 
- the sister PR is already indicated in the OPS section : https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/938
- After the input by @YuryHrytsuk , this PR does NOT change any docker-compose anymore. For the moment I am just adding the scripts and the env vars but these are for the moment not applied
- I will follow up with ops what would be the most systematic way to create/update and delete this user.

Reply:
ok, let's not forget about it :--)

Reply:
After doing some reading I find unsafe to add this in the migration. Will follow up with some automatitions in the future but for the moment I will do it "on demand"